src/components/CompareProject.vue

```diff
@@ -111,6 +118,10 @@ export default defineComponent({
         this.taskId = this.tasks[this.taskIndex].taskId
       }
     },
+    createInformationPages,
+    createFallbackInformationPages() {
+      return undefined
```

Comment:
this just means that there are no fallback information pages defined for this project type? It might be worth adding this as as comment here.

Reply:
There are no fallback information pages for Compare projects in the app, so neither should we in the web app. I've added a comment here.